test/buddy/auth/backends/token_tests.clj

```diff
@@ -43,6 +44,9 @@
 (def jws-backend (backends/jws {:secret jws-secret}))
 (def jws-backend-with-authfn (backends/jws {:secret jws-secret :authfn (constantly ::jws-authorized)}))
 (def jws-data {:userid 1})
+(def rsa-privkey (keys/private-key "test/_files/privkey.rsa.pem"))
+(def rsa-pubkey (keys/public-key "test/_files/pubkey.rsa.pem"))
+(def jws-backend-rsa (backends/jws {:secret rsa-pubkey}))
```

Comment:
As I have explained in the issue, this line tries initialize the backend with `:hs256` algorithm but you are passing to it the rsa pubkey, that is wrong. If you want use rsa keys, you need pass an appropriate algorithm.